ponylang-mode.el

```diff
@@ -142,6 +147,72 @@
     "while" "with")
   "Pony keywords which indicate a new indentation level.")
 
+(defconst ponylang-declaration-keywords
```

Comment:
are these new consts used?

Reply:
That is a process，It is now in a complete state